src/main/java/hal/exceptions/StartUpException.java

```diff
@@ -0,0 +1,10 @@
+package hal.exceptions;
+
+/**
+ * The `StartUpException` class represents an exception related to startup issues in the Duke application.
+ */
+public class StartUpException extends HalException {
```

Comment:
Might be a bit funny but I find the name misleading since I related to the entrepreneurial startup instead of the process of starting up the application. Would it be better to use `launchException` instead (assuming that you're not a rocket science enthusiast)?